### ISSUE TITLE
Kinomi v2 OV2640 camera interface

### ISSUE THREAD

**Issue by Xerusen** [bigtreetech/KNOMI#49]
Is there any way to use OV2640 interface? kinomi 2 was advertised to be able to use camera and humidity sensor
currently software doesn't take advantage of that

**Comment by Xerusen** [comment]
There is also lis2dw12 accelerometer mentioned, but no instruction on how to use any of those features

**Comment by Coolbobjoe** [comment]
Following as I also bought this with these interests in mind!

**Comment by Bazeone** [comment]
same here

**Comment by azak1r** [comment]
would love to use my 3do nozzle cam from the knomi... saves on wire runs

**Comment by rebel1985** [comment]
me too

**Comment by dukes4** [comment]
Any update please ?

**Comment by BTopbas** [comment]
Up

**Comment by salbang** [comment]
Up

**Comment by benbender** [comment]
@bigtreetech any news?

**Comment by thejoshuahenry** [comment]
Apparently this person got the camera working: https://github.com/OmeeeChan/Knomi-V2-camera-function/tree/main

**Comment by 500Foods** [comment]
Kinda vague on the details tho 😀

**Comment by thejoshuahenry** [comment]
Indeed... I assume just flash that firmware, connect a compatible OV2640 camera, then... navigate to the Knomi web address and grab the webcam stream. Just guessing - will need to get a camera module to try.

**Comment by 500Foods** [comment]
We'll just take it on faith that some random firmware.bin loaded into a device with WiFi and camera hardware is good to go? I mean, sure, I like it, but it just strikes me as funny. But then again, maybe this is the guy who originally designed it, so who knows! Still funny 😆

**Comment by fsedarkalex** [comment]
You might also want to check out my fork of BTT's repo containing the changes from the mentioned firmware as well as some further research and thinkerying by me:

https://github.com/fsedarkalex/KNOMI/tree/firmware-fsedarkalex

Trust is good, verification is better. Check the code and compile it yourself (while the bin is still available as release) if you want to.
